### ISSUE TITLE
make peer Error :  *** "No Docker in PATH: Check Dependencies." Stop

### ISSUE THREAD

**Issue by iamsaurabhc** [hyperledger-archives/fabric#2401]
![screenshot from 2016-09-09 18-32-07](https://cloud.githubusercontent.com/assets/19235748/18387706/c9382a82-76bb-11e6-91e0-586baa07d9af.png)
I'm getting this error while trying tp `make peer` : 
`*** "No Docker in PATH: Check Dependencies". Stop`
. Docker is already installed and PATH also has the folder containing Docker. Then why?
Please help

**Comment by mychewcents** [comment]
Yes, I have had the same issue. I resolved by restarting the OS and making sure that GO variables are set.

**Comment by kebab-mai-haddi** [comment]
@akarshagarwal2010
I am unable to set the GO variables. Using Ubuntu 16.04, would really apreciate your help.
